src/catalogue/items/catalogueItemsDialog.component.test.tsx

```diff
@@ -876,12 +868,6 @@ describe('Catalogue Items Dialog', () => {
         'Please enter a valid value as this field is mandatory'
       );
 
-      const mandatoryFieldBooleanHelperText = screen.getByText(
-        'Please select either True or False'
-      );
-
-      expect(mandatoryFieldBooleanHelperText).toBeInTheDocument();
```

Comment:
Same here

Reply:
Same reason as above - it is no longer clearable